exercises/clock/src/clock.h

```diff
@@ -1,11 +1,19 @@
 #ifndef CLOCK_H
 #define CLOCK_H
 
+#include<stdbool.h>
+
 #define MAX_STR_LEN (5 + 1)     // "##:##\0"
```

Comment:
Neat trick:

```c
enum
{
  MaxStringLength = strlen("##:##") + 1
};
```

Reply:
Fancy ✨  .
But is it clear enough for beginners what is actually happening?

Reply:
Probably not, but I wanted to share anyway :slightly_smiling_face: